icons/nano/nano-original.svg

```diff
@@ -0,0 +1 @@
l.566 1.748.285-2.308h-.456c-.089 0-.154-.015-.196-.046-.042-.031-.062-.08-.062-.148 0-.074.02-.125.06-.154.04-.029.114-.044.222-.044h.938c.043 0 .08.019.113.057s.049.085.049.141c0 .066-.018.115-.055.147-.037.032-.092.048-.165.048h-.035l-.412 2.903c-.006.06-.017.1-.033.119-.016.019-.043.029-.08.029h-.206a.172.172 0 0 1-.102-.026.205.205 0 0 1-.061-.099l-.63-1.998-.557 2.003c-.014.048-.031.08-.052.096-.021.015-.056.023-.105.023h-.218c-.039 0-.067-.009-.086-.028-.018-.018-.031-.06-.039-.126l-.441-2.898h-.035c-.075 0-.132-.016-.17-.049-.038-.033-.057-.081-.057-.145 0-.058.015-.105.046-.142a.142.142 0 0 1 .113-.055h.941c.106 0 .18.015.219.044.04.029.06.08.06.154 0 .068-.021.117-.062.148-.042.031-.106.046-.193.046h-.409l.313 2.308zM98.987 124.027a1.06 1.06 0 0 1-.424-.353.879.879 0 0 1-.142-.498c0-.301.107-.547.322-.74.215-.193.49-.289.825-.289.329 0 .603.097.822.29.219.194.328.43.328.709a.948.948 0 0 1-.139.526 1.019 1.019 0 0 1-.433.354.957.957 0 0 1 .45.35.934.934 0 0 1 .157.541c0 .319-.11.579-.33.78-.22.201-.505.302-.855.302s-.635-.102-.854-.305a1.026 1.026 0 0 1-.328-.789c0-.197.052-.373.157-.526a.983.983 0 0 1 .444-.352zm-.125.878c0 .205.064.372.193.499s.3.192.512.192c.215 0 .387-.062.515-.186a.652.652 0 0 0 .193-.494.646.646 0 0 0-.197-.489.712.712 0 0 0-.511-.187.715.715 0 0 0-.508.184.632.632 0 0 0-.197.481zm.029-1.729c0 .192.063.349.19.471a.673.673 0 0 0 .486.183.67.67 0 0 0 .489-.185.634.634 0 0 0 .19-.476.61.61 0 0 0-.19-.461.68.68 0 0 0-.489-.182.672.672 0 0 0-.488.182.626.626 0 0 0-.188.468zM102.555 124.027a1.06 1.06 0 0 1-.424-.353.879.879 0 0 1-.142-.498c0-.301.107-.547.322-.74.215-.193.49-.289.825-.289.329 0 .603.097.822.29.219.194.328.43.328.709a.948.948 0 0 1-.139.526 1.019 1.019 0 0 1-.433.354.957.957 0 0 1 .45.35.934.934 0 0 1 .157.541c0 .319-.11.579-.33.78-.22.201-.505.302-.855.302s-.635-.102-.854-.305a1.026 1.026 0 0 1-.328-.789c0-.197.052-.373.157-.526a.994.994 0 0 1 .444-.352zm-.124.878a.67.67 0 0 0 .193.499c.129.128.3.192.512.192.215 0 .387-.062.515-.186a.652.652 0 0 0 .193-.494.646.646 0 0 0-.197-.489.712.712 0 0 0-.511-.187.715.715 0 0 0-.508.184.629.629 0 0 0-.197.481zm.029-1.729c0 .192.063.349.19.471a.673.673 0 0 0 .486.183.67.67 0 0 0 .489-.185.634.634 0 0 0 .19-.476.61.61 0 0 0-.19-.461.68.68 0 0 0-.489-.182.672.672 0 0 0-.488.182.626.626 0 0 0-.188.468zM106.011 125.904H105.421c-.072 0-.125-.017-.161-.052-.036-.035-.054-.087-.054-.157 0-.075.021-.128.062-.157s.12-.044.237-.044h.052v-2.616h-.046c-.116 0-.196-.015-.24-.046-.044-.031-.065-.085-.065-.163 0-.072.018-.124.054-.158.036-.034.091-.051.167-.051h1.226c.482 0 .866.155 1.153.466.286.311.43.73.43 1.259 0 .257-.041.496-.123.717s-.202.414-.359.581a1.282 1.282 0 0 1-.53.33c-.208.069-.476.103-.803.103-.039 0-.104-.002-.196-.006a5.345 5.345 0 0 0-.214-.006zm0-.409a3.882 3.882 0 0 0 .363.017c.476 0 .825-.106 1.047-.319.222-.213.332-.549.332-1.008 0-.461-.111-.796-.334-1.006-.223-.21-.579-.315-1.068-.315-.091 0-.16.001-.208.003a1.293 1.293 0 0 0-.132.012v2.616z"/></g></svg>
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
We haven't been enforcing this earlier, should we add it to the SVG Standards too?